src/create-config.ts

```diff
@@ -0,0 +1,322 @@
+      authorizers,
+      contracts,
+      id: `${chainId}`,
+      providers,
+      type: 'evm',
+      options,
+      maxConcurrency: 100,
+      blockHistoryLimit: 300,
+      minConfirmations: 0,
+    };
+  });
+
+  const nodeSettings: NodeSettings = {
+    nodeVersion: '0.5.0',
+    cloudProvider: {
+      type: 'aws',
+      region: 'us-east-1',
+      disableConcurrencyReservations: true,
+    },
+    airnodeWalletMnemonic: '${AIRNODE_WALLET_MNEMONIC}',
+    heartbeat: {
+      enabled: response.airnodeHeartbeat,
+      ...(response.airnodeHeartbeat && {
+        apiKey: '${HEARTBEAT_API_KEY}',
+        id: '${HEARTBEAT_ID}',
+        url: '${HEARTBEAT_URL}',
+      }),
+    },
+    httpGateway: {
+      enabled: true,
+      apiKey: '${HTTP_GATEWAY_API_KEY}',
+      maxConcurrency: 20,
+    },
+    httpSignedDataGateway: {
+      enabled: true,
+      apiKey: '${HTTP_SIGNED_DATA_GATEWAY_API_KEY}',
+      maxConcurrency: 20,
+    },
+    logFormat: 'plain',
+    logLevel: 'INFO',
+    stage: 'dev',
+    skipValidation: true,
+  };
+
+  // Push the NodeSetting secrets to the secrets array
+  secrets.push(`AIRNODE_WALLET_MNEMONIC=`);
+  response.airnodeHeartbeat && secrets.push(`HEARTBEAT_API_KEY=`);
+  response.airnodeHeartbeat && secrets.push(`HEARTBEAT_ID=`);
+  response.airnodeHeartbeat && secrets.push(`HEARTBEAT_URL=`);
+  secrets.push(`HTTP_GATEWAY_API_KEY=`);
+  secrets.push(`HTTP_SIGNED_DATA_GATEWAY_API_KEY=`);
+
+  //generate the triggers from the OISes
+  const oisTriggers: Trigger[] = await Promise.all(
+    Object.keys(apiData.ois).flatMap((oisFilename) => {
+      const ois = apiData.ois[oisFilename];
+      return ois.endpoints.map(async (endpoint) => ({
+        endpointId: await deriveEndpointId(ois.title, endpoint.name),
+        endpointName: endpoint.name,
+        oisTitle: ois.title,
+      }));
+    })
+  );
+
+  const triggers: Triggers = {
+    rrp: oisTriggers,
+    http: oisTriggers,
+    httpSignedData: oisTriggers,
+  };
+
+  const apiCredentials: ApiCredentials[] = Object.keys(apiData.ois).flatMap((oisFilename) => {
+    const ois = apiData.ois[oisFilename];
+    return Object.keys(ois.apiSpecifications.components.securitySchemes).map((security) => {
+      // push the security scheme to the secrets
+      secrets.push(`SS_${security.toUpperCase()}=`.replace(/ /g, '_'));
+      return {
+        oisTitle: ois.title,
+        securitySchemeName: security,
+        securitySchemeValue: `\${SS_${security.toUpperCase()}}`.replace(/ /g, '_'),
+      };
+    });
+  });
+
+  const Config: Config = {
+    chains,
+    nodeSettings,
+    triggers,
+    ois: Object.values(apiData.ois),
+    apiCredentials,
+  };
+
+  //// Build Airkeeper.json ////
+
+  const airkeeperChains: AirkeeperChainConfig[] = apiChains.map((chainName) => {
+    const chainId = chainNameToChainId[chainName];
+    //TODO: Add RrpBeaconServer and DapiServer contracts based on chain
+    const RrpBeaconServer = '';
+    const DapiServer = '';
```

Comment:
Currently we don't have the addresses, I think the plan is to have the addresses in this repository ?